src/lib/commandBarConfigs/validators.ts

```diff
@@ -46,6 +47,20 @@ function isSelections(selections: unknown): selections is Selections {
   )
 }
 
+function parseEngineError(engineError: string) {
```

Comment:
Should add a unit test for this guy

Reply:
@nadr0 as discussed in the call this is the cheap and isolated way of getting this to work here for our case

Reply:
@nadr0 I went in the rabbit hole of rejecting a typed object instead but quickly got out cause it really felt like one. Let me know if you got any thoughts on this